<launch>
    <!-- 加载参数 -->
    <rosparam command="load" file="$(find dwa_navigation)/config/dwa_params.yaml"/>

    <include file="$(find rosbridge_server)/launch/rosbridge_tcp.launch">
        <arg name="bson_only_mode" value="true"/>
    </include>
    
    <!-- 启动DWA节点 -->
    <node pkg="dwa_navigation" type="dwa_planner" name="dwa_planner" output="screen"/>
    
    <!-- 可视化工具 -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find dwa_navigation)/save.rviz"/> 
    
</launch>